<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bienes</class>
 <widget class="QWidget" name="Bienes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bienes</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro de bienes</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>162</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblPlaca">
      <property name="text">
       <string>Placa</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtPlaca"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNombreBien">
      <property name="text">
       <string>Nombre del Bien</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtNombreBien"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblCategoria">
      <property name="text">
       <string>Categoria</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtCategoria"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblDescripcion">
      <property name="text">
       <string>Descripción</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtDescripcion"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblestado">
      <property name="text">
       <string>Estado:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkEstado">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Asignable</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkEstado_2">
      <property name="text">
       <string>Exclusión</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="checkEstado_3">
      <property name="text">
       <string>Reparación</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>256</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/guardar.jpg</normaloff>recursos/guardar.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/editar.jpg</normaloff>recursos/editar.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/eliminar.jpg</normaloff>recursos/eliminar.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/cancelar.png</normaloff>recursos/cancelar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>recursos/departamento.png</normaloff>recursos/departamento.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
